src/test.js

```diff
@@ -0,0 +1,76 @@
+import React from "react";
+import firebase from "firebase/app";
+import "firebase/firestore";
+import fb from "./lib/firebase.js";
+
+class Items extends React.Component {
+  constructor() {
+    super();
+    this.state = {
+      itemName: "",
+      dbItems: []
+    };
+  }
+
+  captureInput = e => {
+    this.setState({
+      [e.target.name]: e.target.value
+    });
+    console.log(this.state);
+  };
+
+  addItem = e => {
+    e.preventDefault();
+    let db = firebase.firestore();
+    // db.settings({
```

Comment:
Same q here with the comments -- if I was coming into this code and working on it, I wouldn't know if it's safe to delete. If it's not, I'd like to read an explanation as to why.